React/Views/ScrollView/RCTScrollContentView.m

```diff
@@ -51,13 +51,12 @@ - (void)reactSetFrame:(CGRect)frame
   // the contents will overflow causing the scroll indicators to appear unnecessarily.
   NSScrollView *platformScrollView = [scrollView scrollView];
   if ([platformScrollView scrollerStyle] == NSScrollerStyleLegacy) {
-    CGFloat horizontalScrollerHeight = [platformScrollView hasHorizontalScroller] ? NSHeight([[platformScrollView horizontalScroller] frame]) : 0;
+    BOOL hasHorizontalScrollerAndNotEmpty = [platformScrollView hasHorizontalScroller] && platformScrollView.documentView.frame.size.height > 0;
+    CGFloat horizontalScrollerHeight = hasHorizontalScrollerAndNotEmpty ? NSHeight([[platformScrollView horizontalScroller] frame]) : 0;
     CGFloat verticalScrollerWidth = [platformScrollView hasVerticalScroller] ? NSWidth([[platformScrollView verticalScroller] frame]) : 0;
 
-    RCTScrollContentLocalData *localData =
-      [[RCTScrollContentLocalData alloc]
-      initWithVerticalScrollerWidth:horizontalScrollerHeight
-           horizontalScrollerHeight:verticalScrollerWidth];
```

Comment:
These were reversed. However since we enable both scroll indicators on init, they always had the same value. 

https://github.com/microsoft/react-native-macos/blob/4299a8510615fa0b8d41c2ed1d5d77b94842071e/React/Views/ScrollView/RCTScrollView.m#L70-L71